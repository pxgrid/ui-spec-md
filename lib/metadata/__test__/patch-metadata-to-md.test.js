const patchMetadataToMd = require('../patch-metadata-to-md')

const md = `---
title: タイトル
screen: ./screen.png?highlight=[[1,2,3,4],[5,6,7,8]]
---

# 見出し1
## 見出し2
### 見出し3
`
const metadataToPatch = {
  screen: './patched-screen.png?highlight=[[11,12,13,14],[15,16,17,18]]',
}

test('Markdown metadata should be updated correctly with patch data.', () => {
  expect(patchMetadataToMd(metadataToPatch, md)).toMatchSnapshot()
})
