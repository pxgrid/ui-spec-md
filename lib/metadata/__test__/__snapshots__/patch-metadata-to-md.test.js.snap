// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Markdown metadata should be updated correctly with patch data. 1`] = `
"---
title: タイトル
screen: ./patched-screen.png?highlight=[[11,12,13,14],[15,16,17,18]]
---

# 見出し1
## 見出し2
### 見出し3
"
`;
